### DISCUSSION TITLE
Welcome to XiaoMi-Pro-Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by daliansky** [daliansky/XiaoMi-Pro#513] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarFre22** [comment]
Hi everyone!

Thank you very much for your hard work! Every time I saw a new release my eyes shine! Hope our community grows even more with the current supported devices but also with new xiaomi's (despite not thinking about it at the time (when I purchased my faithful companion) I'm lucky to have bought the 2017 version due to your excellent support).

@daliansky , answering your question, I'm a tech enthusiast, and I was an old Windows user, due to the classic windows annoying/stupid bugs I tried some Linux Distros. Unfortunately, I was not happy with it, so I gave MacOS a try and I liked it. This was only possible with your project and the help of this extraordinary community.

Sorry to ask, is there any automatic translator for github issues we can use? because it's not very user-friendly to translate manually the Mandarin/Chinese ones.

I think Github Discussions is a nice place to have informal and quick conversations and support, thanks @daliansky ;)
Hope more people join this talk 😃

**Comment by stevezz3** [comment]
@MarFre22 Thank you for the nice words. daliansky is busy as he has to maintain his blog and Hackintosh collection, so I am in charge of maintaining this repository all the time. I will transfer your compliments to him.

> Sorry to ask, is there any automatic translator for github issues we can use? because it's not very user-friendly to translate manually the Mandarin/Chinese ones.

For now, I recommend using Google Translator or using the translator embedded in Chrome for those Chinese issues. By the way, most of the important notes and messages in this repository are written in English, so don't need to worry about missing anything critical ^^.

**Reply by MarFre22** [reply]
@stevezhengshiqi , Also thank you very much for maintaining this project!

> For now, I recommend using Google Translator or using the translator embedded in Chrome for those Chinese issues. By the way, most of the important notes and messages in this repository are written in English, so don't need to worry about missing anything critical ^^.

All right, thanks 😁

**Comment by cook1e0** [comment]
Also wanted to say a quick thank you to @stevezhengshiqi for all your time maintaining this repo despite not even owning this device anymore. Much appreciated.

**Reply by stevezz3** [reply]
You are so welcome.

**Comment by msr1715** [comment]
Didn't wanted to put that on the issue section, I saw that there are recently 2 distinct releases KBL and CML, what does that mean?
Kaby Lake and Comet Lake?

**Reply by stevezz3** [reply]
Yes, you are right. https://github.com/daliansky/XiaoMi-Pro-Hackintosh#identify-your-model

**Comment by louloukou** [comment]
hi every one i'm very excited to read all about the hackintosh your work here is enormous, i didn't saw anything about the redmibook 14 and 15 Pro and about the notebook 14 and 15 Pro so i would like to know if it's possible to make the hackintosh for those models (i would like to buy the redmibook 1 pro) 

thanks a lot and have a  nice day
